cmd/subcommands/init.go

```diff
@@ -0,0 +1,10 @@
+/*
+The purpose of this file is to initialize all of the commands, that are spread across different folders,
+and to keep the imports in one place, keeping main.go clean.
+*/
```

Comment:
I don't think this is improvement compared to keeping imports in `main.go`. I would not consider imports a clutter in `main.go` (after all `cmd` import is effectively the same type of import as any other subcommand).

Reply:
still vote for this approach. 
i will leave all of the imports over here and move to main at the very end, if needed